index.bs

```diff
@@ -1036,6 +1036,10 @@ An <dfn>attribution rate-limit record</dfn> is a [=struct=] with the following i
 :: A [=moment=].
 : <dfn>expiry time</dfn>
 :: Null or a [=moment=].
+: <dfn>entity ID</dfn>
+:: Null or an ID.
```

Comment:
What is an `ID`?

Reply:
This PR is to be rebased on https://github.com/WICG/attribution-reporting-api/pull/1211 and this is just a placeholder.